src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.serviceimpl.DataConverterImpl;
+import core.basesyntax.serviceimpl.ReaderServiceImpl;
+import core.basesyntax.serviceimpl.ReportGeneratorImpl;
+import core.basesyntax.serviceimpl.ShopServiceImpl;
+import core.basesyntax.serviceimpl.WriterServiceImpl;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+        // 1. Read the data from the input CSV file
+        ReaderService<List<String>> readerService = new ReaderServiceImpl();
+        List<String> inputReport = readerService.readFromFile("src/main/java/resources/input.csv");
```

Comment:
The file path should not be hardcoded in services. Instead, pass the file path as an argument to the main method and use it here.

Reply:
make a constant out of the path